docs/guidelines-general.md

```diff
@@ -0,0 +1,58 @@
+# General Guidelines
+
+This document contains general guidelines and useful references related to the review process.
+
+We're still in the early stages of the process, trying to figure out how this should (could) work.
+This is a live document and we should expect it to be updated as time goes on.
+
+## What is a code review?
+
+TBC
+
+## What is a cross-project code review?
+
+TBC
+
+## The process
+
+There's no strict process that reviewers and reviewees are required to go through: this is supposed to be flexible and allow developers to choose the approach that best suits them and their project.
+Nevertheless it may be helpful to have some understanding about how we've done this in the past, what worked and what didn't.
+
+Please try to avoid making this section prescriptive.
+
+The high-level approach we've used up until now has been.
+
+1. Create a personal fork of this [`the-reg-review`](https://github.com/alan-turing-institute/the-reg-review) repository and create a new branch in it for your work.
+2. Add a new subdirectory `<date>-<name>` to the working tree for this new branch and add all of the code to be reviewed to it.
+3. Commit these changes to your personal fork.
+4. Create a Pull Request from the new branch on your personal fork to `main` in the upstream repo.
```

Comment:
I just made a branch, didn't fork. It does presume that the reviewee has push access to the repo, but I think as long as the reviewee is in hut23 that should be fine?

Reply:
Yes, that's something I was thinking about and I appreciate you didn't fork, but you have the privilege of being a repo admin!

My personal preference would be to avoid giving everyone write access and to ask people to fork, since that's a more typical workflow for open source projects. But I know I'm exposing personal bias by saying that.

Reply:
Oh! I didn't know I'm an admin 😳 I thought I just had write access via the Hut23 team 😄

I think for REG stuff I've usually seen that the Hut23 github team is granted write permissions, which is why I assumed that was the case. I think the benefit of that is that it's more convenient (no need to fork) and also a bit more future-proof as it doesn't rely on a select group of people with write access; but this is just my perspective and I don't know what I don't see, so it's up to you how you'd prefer to handle this :)

On a completely separate note, I wonder if it would be a good idea to include a PR template in `.github/PULL_REQUEST_TEMPLATE` with a suggested structure / checklist for reviewees (which perhaps includes 'read this document')?

Reply:
Yup, I can do that after this PR is merged! Then there will be something to link to